src/components/Pagination/Pagination.tsx

```diff
@@ -113,6 +113,9 @@ export const Pagination: VFC<Props & ElementProps> = ({
 
 const Wrapper = styled.nav`
   display: inline-block;
+  max-width: 100%;
+  overflow-x: auto;
+  vertical-align: top;
```

Comment:
> inline 要素に overflow を設定すると下側に余分なスペースが生じてしまう

そもそも `inline-block` である理由がないように見えました。

```suggestion
  max-width: 100%;
  overflow-x: auto;
```